jupyter-book/spatial/neighborhood.ipynb

```diff
@@ -0,0 +1,586 @@
+{
```

Comment:
<p>You're not inspecting it in the line below.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/cross-modal-single-cell-best-practices/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>